ORStools/gui/ORStoolsDialog.py

```diff
@@ -446,7 +446,7 @@ def _linetool_annotate_point(self, point, idx):
         annotation.setFrameSizeMm(QSizeF(7, 5))
         annotation.setFrameOffsetFromReferencePointMm(QPointF(1.3, 1.3))
         annotation.setMapPosition(point)
```

Comment:
Shouldn't the point also be transformed like in the `_on_linetool_map_click` function?

```
transformer = transform.transformToWGS(map_crs)
point_wgs = transformer.transform(point)
```

Reply:
Cool. Wasn't aware of the sequence the functions are called. If no coordinates reach `_linetool_annotate_point` without being transformed, forget my comment ;).